app/controllers/HomeController.scala

```diff
@@ -84,6 +95,20 @@ class HomeController @Inject()(db: Database, cc: ControllerComponents, service:
     service.close(queueId)
   }
 
+  def queues = Action.async {
+    val patient = new Patient(2, "Foo", "Bar")
+    val result = for {
+      queues <- getMockQueue(patient)
+    } yield {
+      Ok(views.html.queues(queues))
+    }
+
+    result.recover {
```

Comment:
Is it necessary?

Reply:
When using `Action.async` it seems to be necessary